docroot/modules/humsci/hs_migrate/hs_migrate.module

```diff
@@ -31,6 +31,19 @@ function hs_migrate_help($route_name, RouteMatchInterface $route_match) {
   }
 }
 
+/**
+ * Migrate callback to get the current time.
+ *
+ * @param mixed $arg
+ *   Argument passed from the callback migrate process plugin.
+ *
+ * @return int
+ *   Current time.
+ */
+function _hs_migrate_get_time($arg = NULL) {
+  return time();
+}
```

Comment:
Why is this needed?

Reply:
i'll document like i did in the cardinal service profile
see https://github.com/SU-HKKU/cardinal_service_profile/blob/8.x-1.x/cardinal_service_profile.profile#L36

> We use this function in migration callback processes because using `time` as the callback produces messages about "function accepts 0 arguments, 1 argument passed". So we just have our own callback that takes the argument from the migration process and does nothing with it.

Reply:
Thanks.